ambry-router/src/test/java/com.github.ambry.router/GetBlobOperationTest.java

```diff
@@ -163,7 +163,7 @@ public void after() {
   @Parameterized.Parameters
   public static List<Object[]> data() {
     return Arrays.asList(
-        new Object[][]{{SimpleOperationTracker.class.getSimpleName(), false}, {AdaptiveOperationTracker.class.getSimpleName(), false}});
+        new Object[][]{{SimpleOperationTracker.class.getSimpleName(), false}, {SimpleOperationTracker.class.getSimpleName(), true}, {AdaptiveOperationTracker.class.getSimpleName(), false}, {AdaptiveOperationTracker.class.getSimpleName(), true}});
```

Comment:
need only one of the two cases for encrypted = true (either simple or adaptive). I don't believe both are useful - what do you think?

Reply:
Ok to use one. 
I prefer AdaptiveTacker which fully go though the code by `super()` and it's used in our prod.